src/screens/SearchScreen/components/DictionaryEntry.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import PropTypes from 'prop-types';
+
+const DictionaryEntry = props => {
+  return (
+    <View>
+      <Text>{props.updateInfo(props.entryNum)}</Text>
+    </View>
+  );
+};
+
+DictionaryEntry.propTypes = {
+  data: PropTypes.isRequired,
```

Comment:
It looks like this isn't the correct proptype. From what I can see, it looks like you're passing down an `entryNum` and an `updateInfo` into the entry, but all `updateInfo` does is verify the data shape and display an error otherwise — I _think_ it would make more sense to pass the entire `entry` in, and put the logic currently in `updateInfo` into this component instead, but I could be missing something.